server/service/src/invoice_line/stock_out_line/update/generate.rs

```diff
@@ -10,6 +13,7 @@ pub fn generate(
     item_row: ItemRow,
     batch_pair: BatchPair,
     invoice: InvoiceRow,
+    pricing: ItemPrice,
```

Comment:
sell price would already be set in update, can we use existing line sell price here please

Reply:
Added to follow up - https://github.com/msupply-foundation/open-msupply/issues/4763